tendrl/commons/manager/rpc_job_process.py

```diff
@@ -3,26 +3,21 @@
 import traceback
 import uuid
 
-import etcd
 import gevent.event
 import yaml
 
 from tendrl.commons.definitions.validator import DefinitionsSchemaValidator
 from tendrl.commons.flows.exceptions import FlowExecutionFailedError
+from tendrl.commons.utils import import_utils
 
 LOG = logging.getLogger(__name__)
 
 
 class EtcdRPC(object):
-    def __init__(self, syncJobThread):
-        self.config = syncJobThread._manager._config
-        etcd_kwargs = {
-            'port': int(self.config.get("commons", "etcd_port")),
-            'host': self.config.get("commons", "etcd_connection")
-        }
-
-        self.client = etcd.Client(**etcd_kwargs)
-        self.syncJobThread = syncJobThread
+    def __init__(self, syncjobthread, etcd_client):
```

Comment:
Below, the creation code for EtcdRpc is not passing etcd_client to the constructor.